README.md

```diff
@@ -896,6 +896,16 @@ Note that the Circus Train configuration only supports one set of AWS credential
           table-name: aws-a_table
           table-location: s3://<aws-a-data-bucket>/<pata-to-aws-a_db>/aws-a_table
 
+## Circus Train extensions
```

Comment:
Could you move this instead to be the final sub-section under "Extension Points" as I think it fits in more logically there. So I'd suggest a `###` level sub-heading and we should call it "External Extensions" instead to be clear that these are outside of Circus Train itself.

Reply:
Thanks for your comment. It has been added now.